### PR TITLE
feat: digital business card

### PR DESCRIPTION
**Pull request by amanji**
*Issue #:* bcgov/entity#18284 + bcgov/entity#18394

*Description of changes:* See PR #552

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: .github/workflows/legal-api-ci.yml
Comment (severinbeauvais) [inline .github/workflows/legal-api-ci.yml:70]: Do we have the BUSINESS_SCHEMA_ID and BUSINESS_CRED_DEF_ID values for Test and Prod yet? If so, please email them to me and I can put them in 1Password. If not, how do we get them?

Comment (severinbeauvais) [reply .github/workflows/legal-api-ci.yml:70]: Also, there are 4 TRACTION_xxx keys that are not yet configured in Test and Prod.

**Review by severinbeauvais — APPROVED** [review]
Wait for Argus' review before merging.

**Review by argush3 — APPROVED** [review]

**Pull request merged**
